gradle-enterprise-conventions-maven-extension/extension/pom.xml

```diff
@@ -0,0 +1,157 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.example</groupId>
+    <artifactId>gradle-enterprise-conventions-maven-extension</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Gradle Enterprise Conventions Maven Extension</name>
+    <description>A Maven extension to enable and configure Maven Build Scans in Gradle Enterprise</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-core</artifactId>
+            <version>3.9.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-component-annotations</artifactId>
+            <version>2.1.1</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.gradle</groupId>
+            <artifactId>gradle-enterprise-maven-extension</artifactId>
+            <version>1.16.5</version>
+        </dependency>
+        <dependency>
+            <groupId>com.gradle</groupId>
+            <artifactId>common-custom-user-data-maven-extension</artifactId>
```

Comment:
This doesn't work correctly. The result is a scan like [this](https://ge.solutions-team.gradle.com/s/iqvfczktee7xg), where the tags, links and values we expect from CCUD are not present. The CCUD Listener does not appear in [Extensions](https://ge.solutions-team.gradle.com/s/iqvfczktee7xg/extensions?expanded=WyIxIiwiNCIsIjExIiwiMCIsIjEwIl0#4), despite being declared here as a [compile dependency](https://ge.solutions-team.gradle.com/s/iqvfczktee7xg/dependencies?toggled=WyIwLjAiLCIwLjAtMS4wIl0#MC0wLTAtMw)